README.rst

```diff
@@ -87,7 +87,7 @@ Inside tests it is possible to change the log level for the captured
 log messages.  This is supported by the ``caplog`` funcarg::
 
     def test_foo(caplog):
-        caplog.setLevel(logging.INFO)
+        caplog.set_level(logging.INFO)
```

Comment:
The API was changed long ago: in f5c3537d70a2d95303f71e7bea4b96878a1887b1 (and tests in 948bfcf5ba6fe9969c66e06a76d9a1f3900683ca), I've only updated the docs to be in-sync with the code.

Let's consider restoring backward compatibility as a separate issue. :cactus: